scripts/compile.sh

```diff
@@ -48,7 +48,11 @@ DOTNET_PATH=$(which dotnet)
 PREFIX="$(cd -P "$(dirname "$DOTNET_PATH")/.." && pwd)"
 
 header "Restoring packages"
-$DNX_ROOT/dnu restore "$REPOROOT" --quiet --runtime "$RID" --no-cache
+$DNX_ROOT/dnu restore "$REPOROOT/src" --quiet --runtime "$RID" --no-cache
```

Comment:
Need to escape the error during restore _testapp_. There are projects in _testapp_ intentionally have broken dependencies for test. The restore failure on it shouldn't fail the build.